avidemux_fr.ts

```diff
@@ -9772,7 +9772,7 @@ L&apos;édition en cours a été enregistrée et sera rechargée au prochain dé
     <message>
         <location line="+176"/>
         <source>Restore previous session</source>
-        <translation type="unfinished"></translation>
+        <translation>Restaurer la version précédente</translation>
```

Comment:
My French is severely limited, but I would expect "Restore previous session" to amount to "Restaurer la séance précédente". Are you sure "version" can be used to describe the period of time a user interacts with an application before closing it?

Actually, the name of the function is slightly misleading as it just restores the final state of editing of the last video loaded, but oh well...

Reply:
You right ! It is a mistake. I create a new pull request with the changes.